packages/@webex/plugin-meetings/src/roap/index.ts

```diff
@@ -209,7 +212,41 @@ export default class Roap extends StatelessWebexPlugin {
             meeting.updateMediaConnections(mediaConnections);
           }
 
-          return locus;
+          let roapAnswer;
+
+          if (mediaConnections?.[0]?.remoteSdp) {
```

Comment:
Should we do something when mediaConnections[0]?.remoteSDP is undefined?

I think this might be handled elsewhere, so a comment may be nice to have here for the cases when this would be undefined.

Reply:
I think it would indicate a malformed response from the server in this case at this stage, right?

Reply:
I think mediaConnections[0]?.remoteSDP is always there, I've added the code to handle it missing just in case, I don't think we should ever get a 2xx response from Locus and that not be there.